frappe/printing/doctype/print_format/print_format.json

```diff
@@ -13,6 +13,7 @@
   "standard",
   "custom_format",
   "disabled",
+  "chrome_pdf_generator",
```

Comment:
```suggestion
  "pdf_generator",
```

This could be "select" field type, default value should be "WKHTML2PDF"... "CHROME" option should come from your app via property setter.

example: https://github.com/frappe/exotel_integration/blob/e124a6077b58d80d31ac54f2963ba22b81f6791b/exotel_integration/utils/utils.py#L5